DESCRIPTION

```diff
@@ -18,6 +18,7 @@ Suggests:
     aws.s3,
     uuid,
     R.utils,
-    testthat
+    testthat,
+    urltools
```

Comment:
@alondhe : would you mind adding this new dependency here as well?: https://github.com/OHDSI/DatabaseConnector/blob/master/R/InsertTable.R#L260